app/assets/javascripts/directives/courseForm.js

```diff
@@ -90,14 +104,14 @@ planner.directive('courseForm', ['Restangular', '$timeout', 'courseService', 'te
       scope.checkFormValidity = function() {
         scope.continuousSessions = checkSessionsContinuous();
         var validFields = checkValidFields();
-        scope.courseForm.$valid = validFields && !!scope.courseParams.session_ids.length && scope.continuousSessions;
+        scope.courseForm.$valid = validFields && scope.continuousSessions;
         scope.courseForm.$invalid = !scope.courseForm.$valid;
       };
 
       var checkSessionsContinuous = function() {
         var sessions = scope.courseParams.session_ids;
         if (!sessions.length) {
-          return false;
+          return true;
         } else if (sessions.length === 1) {
           return true;
         }
```

Comment:
This is a funny little ditty. If sessions are no longer required, we can remove this function. :stuck_out_tongue: